app/src/main/java/com/example/polina/socialnetwork/PostAdapter.java

```diff
@@ -41,6 +46,7 @@
     ImageLoader mImageLoader;
     RequestQueue queue;
     ViewHolder holder;
+    SNApp snApp = new SNApp();
```

Comment:
never initialise new instance of Android Application class. Never!!!
you can get instance from context: snApp = (SNApp)context.getApplicationContext();